server/chat-plugins/scavengers.ts

```diff
@@ -1927,7 +1927,7 @@ const ScavengerCommands: ChatCommands = {
 		const ladder = await Leaderboard.visualize('points') as AnyObject[];
 		this.sendReply(
 			`|uhtml${isChange ? 'change' : ''}|scavladder|<div class="ladder" style="overflow-y: scroll; max-height: 300px;"><table style="width: 100%"><tr><th>Rank</th><th>Name</th><th>Points</th></tr>${ladder.map(entry => {
-				const isStaff = room!.auth.has(toID(entry.name));
+				const isStaff = ['+', '%', '@', '*', '#'].includes(room!.auth.getDirect(toID(entry.name)));
```

Comment:
```suggestion
				const roomRank = room!.auth.getDirect(toID(entry.name));
				const isStaff = Users.Auth.atLeast(roomRank, '+');
```
Bit cleaner and accounts for other ranks that may be added.